src/py42/sdk/queries/__init__.py

```diff
@@ -26,3 +27,16 @@ def any(cls, *args):
     @classmethod
     def all(cls, *args):
         return cls(*args)
+
+
+class Choices:
```

Comment:
Since this gets used outside of the search queries, it should probably live in a different file, perhaps `util.py` or its own file `choices.py` at the root of the repo

Reply:
Probably it is own file, the more I think about this. Then, we don't have to include it in the generated-docs and it can be just an implementation detail that we can freely change.